### PR TITLE
APP-4579 Condensed mode - Button component

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Description 

- Apply condensed mode to the button component

### Demo
Screenshot 2021-11-08 at 09 28 37

### CODE REVIEW

File: src/utils/mixins/_buttons.scss [hunk 1]
```diff
@@ -81,3 +83,50 @@
   }
   @include focus-visible($colorMap, true);
 }
+
+/* Button sizes */
+@mixin condensedButton {
+  .condensed {
+    :not(.tk-scape-condensed *) {
```
Comment (AliciaSymphony) [inline src/utils/mixins/_buttons.scss:90]: "escape"

**Review by AliciaSymphony — COMMENTED** [review]
Cool!
One suggestion
- It would be nice to also document the small and see explicitly that the design is unchanged

Other comment out of scope
- Would be so nice to have the condensed button directly at the "dark" mode level (found this: https://storybook.js.org/addons/storybook-addon-themes)
image

**Review by AliciaSymphony — COMMENTED** [review]
Now I am thinking, dynamic examples is very nice for the documentation!
But this will be skipped by the Visual Testing

**Comment by symphony-annaLlorens** [comment]
@AliciaSymphony Thanks for the review.
They are on the condensed mode by default, so they can be tested by the visual Testing library.
Very good idea, I will create a story to apply directly condensed mode from the storybook..

**Comment by AliciaSymphony** [comment]
FYI https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/170, no need to create a ticket :)

File: src/utils/mixins/_buttons.scss [hunk 2]
```diff
@@ -81,3 +83,50 @@
...
+  font-size: toRem(12);
+  @include iconSizes(toRem(32), toRem(16), toRem(8));
+}
+@mixin largeButton {
+  padding: toRem(12) toRem(24);
+  height: toRem(40);
+  font-size: toRem(14);
+  @include iconSizes(toRem(40), toRem(16), toRem(8));
+}
+@mixin iconSizes($size, $font-size, $margin) {
```
Comment (symphony-jean-michael) [inline src/utils/mixins/_buttons.scss:118]: Could you please verify that the Button in the loading state is still working? It seems to be a missing case in our Storybook examples in Styles. (T obe sure we don't introduce any regression in components). Thank you :)

Comment (symphony-annaLlorens) [reply src/utils/mixins/_buttons.scss:118]: yes :)

File: stories/button.stories.js [hunk 3]
```diff
@@ -84,18 +115,71 @@ export const Sizes = () => `
...
+    <div class="flex-col">
+      <button class="tk-button tk-button--primary tk-button--large">Button</button>
+    </div>
+  </div>
 </div>
-
+</div>
+<script>
+  var scape = true;
+  function scapeCondensed() {
```
Comment (AliciaSymphony) [inline stories/button.stories.js:171]: "escape"

**Review by AliciaSymphony — APPROVED** [review]
small typo but looks good

**Review by symphony-jean-michael — APPROVED** [review]

**Pull request merged**